impeller/renderer/backend/vulkan/render_pass_vk.cc

```diff
@@ -31,10 +31,8 @@ namespace impeller {
 
 static vk::AttachmentDescription CreateAttachmentDescription(
     const Attachment& attachment,
-    const std::shared_ptr<CommandBufferVK>& command_buffer,
-    bool resolve_texture = false) {
-  const auto& texture =
-      resolve_texture ? attachment.resolve_texture : attachment.texture;
+    const std::shared_ptr<Texture> Attachment::*texture_ptr) {
```

Comment:
What is this syntax?

Reply:
That's a pointer to a field.  It allows us to specify if we want to grab `Attachment::texture` or `Attachment::resolve_texture` instead of using an optional bool argument.  The pointers are const static.  Its easier to check out the calling site.